### DISCUSSION TITLE
Add indeterminate on Checkbox component

### DISCUSSION THREAD

**Discussion by KumJungMin** [primefaces/community#1461] [PrimeVue] [answered]
### Suggest
- According to mdn docs, indeterminate is established and works across many devices and browser versions. It’s been available across browsers since January 2020. ([mdn docs link](https://developer.mozilla.org/en-US/docs/Web/CSS/:indeterminate))

<img width="600" alt="스크린샷 2024-03-19 오전 10 36 27" src="https://github.com/primefaces/community/assets/37934668/fa018a58-9ed6-47b8-ab5e-2ef6ff75c24d">

<br/><br/>

- The indeterminate refers to the state of being neither checked nor unchecked. The image below is an example of indeterminate.
- Because it officially provides indeterminate, Developers expect checkboxes to provide indeterminate option.
- So, i suggest indeterminate option on Checkbox component.
- Thank you for read this suggestion. have a nice day :)
<img width="1028" alt="스크린샷 2024-03-19 오전 10 32 13" src="https://github.com/primefaces/community/assets/37934668/63f35510-7435-4af5-aae6-a69573864935">

**Comment by tugcekucukoglu** [comment, marked as answer]
The version 4.0.0.beta-1 will be released with indeterminate property.

Thank you!

**Reply by KumJungMin** [reply]
Thank you! 😄 Have a nice day
